api/v1/models/post.py

```diff
@@ -24,3 +24,17 @@ class Post(AbstractBaseModel):
 
     def __str__(self) -> str:
         return self.content or self.image or self.video
+
+
+
+class Like(AbstractBaseModel):
+    __tablename__ = "like"
+
+    user_id: Mapped[UUID4] = mapped_column(ForeignKey("user.id"))
```

Comment:
User `id` column may not necessarily be a uuid4 type. Any string is acceptable but it has to be unique.